content/ri/release-notes/v1.13.0.md

```diff
@@ -6,6 +6,19 @@ description: RedisInsight v1.13.0
 weight: 4
 ---
 
+## 1.13.1 (November 2022)
+
+This is the maintenance release of RedisInsight 1.13 (v1.13.1)!
+
+### Fixes:
+- Core:
+  - Fixed container vulnerabilities.
+  - Disable healthcheck API to overload RedisInsight DB.
```

Comment:
Mind explaining this fix a bit further? What was happening?

Reply:
done